DEVELOPMENT.md

```diff
@@ -43,12 +43,25 @@ make sure only properly formatted and lint-free code lands into master.
 
 ### Running Upstream
 
-On macOS you'll have to install some external dependencies, to be able to
-compile everything:
+You'll have to install some external dependencies to be able to compile
+the proxy as well as the UI.
+
+On macOS:
 ```
 xcode-select --install
 sudo xcodebuild -license
-brew install pkgconfig nettle
+brew install yarn pkgconfig nettle
+```
+
+On Debian/Ubuntu:
+```
+# Install yarn (UI dependency):
```

Comment:
I'd advocate against commands which are specific to certain distros. As it doesn't even help a majority of our core devs. We should just list the dependencies we expect to be present and link to the respective projects which will have guidance for installation. This information will be stale very quickly.

Reply:
Fair point. How about now?